.github/workflows/pr-main_l2.yaml

```diff
@@ -43,6 +43,34 @@ jobs:
           name: buildx-cache
           path: /tmp/buildx-cache
 
+  lint:
```

Comment:
Enabling this here since pr-main_l1 has `- "crates/l2/**"` inside `paths-ignore`